### PR TITLE
[13.0][MIG] html_image_url_extractor: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0

Exclude `fetchmail_incoming_log` + `fetchmail_notify_error_to_sender_test` in `travis.yaml` to OCA/OCB

Please @ernestotejeda   and @joao-p-marques can you review it?

@Tecnativa TT27585

### CODE REVIEW

File: html_image_url_extractor/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2016 Tecnativa - Cristina Martin R.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Image URLs from HTML field",
+    "summary": "Extract images found in any HTML field",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-tools",
+    "author": "Tecnativa, " "Onestein, " "Odoo Community Association (OCA)",
```
Comment (ernestotejeda) [inline html_image_url_extractor/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Onestein, Odoo Community Association (OCA)",
```

**Review by ernestotejeda — APPROVED** [review]

**Review by joao-p-marques — COMMENTED** [review]
LGTM :+1: 

Can you just squash the bot commits please?

**Review by chusamo — APPROVED** [review]
LGTM 
Captura de pantalla de 2021-03-14 11-37-05

**Comment by pedrobaeza** [comment]
@victoralmau can you rebase to see the problem?

**Comment by victoralmau** [comment]
Travis is red because appear now error related to `fetchmail_incoming_log` addon.

**Comment by pedrobaeza** [comment]
Travis is still red

**Pull request merged**